src/scenes/titlescreen.ts

```diff
@@ -1,37 +1,45 @@
-import { Nullable, UUID } from "definitions/utils";
+import { float, Nullable } from "definitions/utils";
 import SessionManager from "managers/SessionManager";
 import Phaser from "phaser";
 import { interactify } from "../utilities/utils";
 import { ScreenHeight, ScreenWidth } from "../core/main";
+import Image = Phaser.GameObjects.Image;
```

Comment:
I never seen this before. 😅 I think it should be better to write "Phaser.GameObjects.Image" down at the type, because there are many different image types.